tools/maxquant/macros.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" ?>
+<macros>
+        <token name="@VERSION@">1.6.3.4</token>
+        <xml name="output" token_format="tabular" token_label="default description" token_name="default">
+                <data format="@FORMAT@" label="@LABEL@ for ${on_string}" name="@NAME@">
+                        <filter>'@NAME@' in output</filter>
+                </data>
+        </xml>
+        <xml name="output_option" token_label="default label" token_name="default">
+                <option value="@NAME@">@LABEL@</option>
+        </xml>
+        <xml name="mod_option" token_value="default">
+                <option value="@VALUE@">@VALUE@</option>
+        </xml>
+        <xml name="modification">
+                <expand macro="mod_option" value="Acetyl (K)"/>
```

Comment:
How did you build those lists? Are the scripts from above, which you removed useful for this? How can we update those long list of modifications easily in the future?

Reply:
I used the init.py script in the tool dir. It reads the modifications.xml and enzymes.xml from maxquant and writes them into macros.xml. After that, the macros.xml doesn't comply with your conventions anymore... i'll try to fix this.

Reply:
Ok, could you also add a README.md file to the repo that describes this procedure please?